unity-client/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarAnimatorLegacy.cs

```diff
@@ -235,7 +235,11 @@ public void BindBodyShape(Animation animation, string bodyShapeType, Transform t
         {
             var animationToId = currentAnimations.Get()[i];
             if (this.animation.GetClip(animationToId.id) == null)
```

Comment:
To avoid error proneness I'd remove the id field altogether. But I guess this is good enough.